src/compounders/slipstream/SlipstreamCompounder.sol

```diff
@@ -251,17 +251,17 @@ contract SlipstreamCompounder is IActionBase {
             // Swap full amount of token0 to token1.
             zeroToOne = true;
             amountOut = SlipstreamLogic._getAmountOut(position.sqrtPriceX96, true, fees.amount0);
-        } else if (position.currentTick <= position.tickLower) {
+        } else if (position.currentTick < position.tickLower) {
```

Comment:
That's what you found out recently that price at tick lower is considered in range, but at tick upper considered out of range right ?

Reply:
Yes